hypothesis-python/src/hypothesis/strategies/_internal/types.py

```diff
@@ -351,10 +360,38 @@ def _networks(bits):
         st.sampled_from(SPECIAL_IPv6_RANGES).map(ipaddress.IPv6Network),
     ),
     os.PathLike: st.builds(PurePath, st.text()),
+    UnicodeDecodeError: st.builds(
+        UnicodeDecodeError,
+        st.just("unknown encoding"),
+        st.just(b""),
+        st.just(0),
+        st.just(0),
+        st.just("reason"),
+    ),
+    UnicodeEncodeError: st.builds(
+        UnicodeEncodeError,
+        st.just("unknown encoding"),
+        st.text(),
+        st.just(0),
+        st.just(0),
+        st.just("reason"),
+    ),
+    UnicodeTranslateError: st.builds(
+        UnicodeTranslateError, st.text(), st.just(0), st.just(0), st.just("reason")
+    ),
+    enumerate: st.builds(enumerate, st.just(())),
+    filter: st.builds(filter, st.just(lambda _: None), st.just(())),  # type: ignore
+    map: st.builds(map, st.just(lambda _: None), st.just(())),  # type: ignore
+    reversed: st.builds(reversed, st.just(())),  # type: ignore
+    classmethod: st.builds(classmethod, st.just(lambda self: self)),
+    staticmethod: st.builds(staticmethod, st.just(lambda self: self)),
+    super: st.builds(super, st.from_type(type)),
```

Comment:
Wow, I was not aware that this will work.

Reply:
I'm not really sure what you'd do with an ["unbound super()"](https://docs.python.org/3/library/functions.html#super) aside from 'return an arbitrary instance of `super`'... but that's all I wanted anyway!